wire-ios-request-strategy/Sources/Object Syncs/Helpers/MessageSyncTests.swift

```diff
@@ -99,6 +99,35 @@ final class MessageSyncTests: MessagingTestBase {
         XCTAssertTrue(self.waitForAllGroupsToBeEmpty(withTimeout: 0.5))
     }
 
+    func test_ItDosntGenerate_SyncingMessage_MLS_Request() {
+        [.v0, .v1, .v2, .v3, .v4].forEach {
+            test_ItDoesntGenerateARequest(apiVersion: $0)
+        }
+    }
+
+    func test_ItDoesntGenerateARequest(apiVersion: APIVersion) {
```

Comment:
```suggestion
    private func internalTest_ItDoesntGenerateARequest(apiVersion: APIVersion) {
```
just to make it clearer it's not a test